esmvaltool/preprocessor/_area_pp.py

```diff
@@ -0,0 +1,110 @@
+"""
+Area operations on data cubes
+
+Allows for selecting data subsets using certain latitude and longitude bounds;
+selecting geographical regions; constructing area averages; etc.
+"""
+import iris
+
+
+# slice cube over a restricted area (box)
+def area_slice(mycube, long1, long2, lat1, lat2):
```

Comment:
Use names that are easier to read, e.g.:
```
def area_slice(cube, longitude, latitude):
```
with `latitude` and `longitude` lists with two items in them, start and end, or
``` 
def area_slice(cube, start_longitude, end_longitude, start_latitude, end_latitude):
```
if you want to be more explicit